docs/UserGuide.md

```diff
@@ -209,6 +209,13 @@ Example:
 
 [<sub>return to table of content</sub>](#table-of-content)
 
+### Exiting the program: `exit`
+Type exit to quit the program, alternatively you can exit by clicking on the :X button at the top right hand corner of the program. <br>
```

Comment:
Don't really unds how the linking thing works but if it works it works hehe

Reply:
@blaketingyu i used intelliJ's auto-complete, but it should follow the whole string excluding symbols